src/ontology/uberon-edit.obo

```diff
@@ -116472,7 +116469,6 @@ property_value: seeAlso https://github.com/obophenotype/uberon/issues/539 xsd:an
 [Term]
 id: UBERON:0006720
 name: pterygoid muscle
-alt_id: UBERON:3010730
```

Comment:
WHy are we losing these axioms?

Reply:
Because these terms are obsolete! Which caused the uberon to be outside of OBO.

Reply:
There's a much bigger issue here in that case!  We should be committed to retaining chains of IDs through obsoletion and merges.  There should have been a step at some point where we converted all alt_ids into class declarations + replaced_by mappings.  Without this, updating legacy annotation sets will become impossible.   This is something we need to keep a close eye on across ontologies formerly or currently in OBO format.

Reply:
What is the purpose here of `alt_id`? I could see that the other way around, you may want to keep a replaced_by annotation.. Is this maybe what was meant here?

Reply:
> So in this case UBERON:3010730 is obsolete.

Ahh - I see it is in Uberon after all and replaced_by is there:

![image](https://user-images.githubusercontent.com/112839/128017845-569c7720-b955-4c56-a910-45a8e829d857.png)

As long as this always happens, it's fine to drop the alt_id (although some legacy software may be affected)

Reply:
BTW - there's nothing in OBO format that prevents axiomatisation AFAIK.  Might be some other pipeline or some weirdness with conversion.